prow/cluster/configmaps/config.yaml

```diff
@@ -150,6 +150,7 @@ presubmits:
 
   istio/test-infra:
   - name: test-infra-presubmit
+    skip_report: true
```

Comment:
This diff introduces the `prow/test-infra-presubmit.sh`, but other PRs on these repos don't yet have the file, and the job would fail, therefore I turned off reporting.

re: notifications, Prow does comment on the PR for test failures.